README.md

```diff
@@ -1,21 +1,22 @@
-## xcsoar-data-content
+# xcsoar-data-content
 
 [![Repository URL checker](https://github.com/XCSoar/xcsoar-data-content/actions/workflows/check_repo_urls.yml/badge.svg)](https://github.com/XCSoar/xcsoar-data-content/actions/workflows/check_repo_urls.yml)
 
 This repository currently maintains all the data necessary to create the [XCSoar](https://xcsoar.org)
 File Manager application's [repository manifest file](http://download.xcsoar.org/repository) as well
 as enable some [XCSoar website](https://xcsoar.org/download/data.html) functionality.
 
-
 This data consists of:
 
 1. `content`: The content itself, like:
-   1. Waypoints
-   2. Airspaces
+    - Waypoints
+    - Airspaces
+
 2. `remote`: URLs to external content, like:
-   1. Waypoints hosted on https://soaringweb.org/, etc.
+    -  Waypoints hosted on https://soaringweb.org/, etc.
```

Comment:
Is this URL supposed to be in `<>` as well considering you put the ones below in these tags?